src/applications/personalization/dashboard/tests/components/DashboardAlert.unit.spec.jsx

```diff
@@ -27,13 +27,22 @@ describe('<DashboardAlert />', () => {
     wrapper.unmount();
   });
 
-  it('should pass aXe check', () =>
```

Comment:
This test started failing with an axe error about the `aria-labelledby` value being invalid. It's definitely not invalid and the axe check passes when using the browser extension on the same html. I tried upgrading axe-core to see if that was the issue, but I still got the bad error. I also didn't see any related GH issues about it. So I'm not sure what's going on, other than I assume it has something to do with JSDom.

Reply:
I wasn't able to get these tested locally, my API instance kept on giving a warning that I couldn't save apps in progress. I'm out tomorrow but please holler at me Monday if this is still an open item.